Dockerfile

```diff
@@ -20,6 +20,11 @@ RUN cp /openmrs_distro/distro/target/sdk-distro/web/openmrs-distro.properties /o
 RUN cp -R /openmrs_distro/distro/target/sdk-distro/web/modules /openmrs/distribution/openmrs_modules
 RUN cp -R /openmrs_distro/distro/target/sdk-distro/web/owa /openmrs/distribution/openmrs_owas
 
+#Include Custom Modules for Namibia Distro
+RUN ls la
```

Comment:
This is failing giving this error **ls: cannot access la: No such file or directory**